src/library/styles/pxToEm.js

```diff
@@ -4,8 +4,20 @@
  */
 import fontSize_base from '../themes/fontSizeBase';
```

Comment:
Related: `fontSize_base` is just doing a replace on `px`.  Shouldn't it just use `parseFloat`?

Reply:
Oh nevermind, this update is coming in #687